libs/membership/website/src/lib/subscribe/subscribe-container.tsx

```diff
@@ -169,6 +174,17 @@ export const SubscribeContainer = <T extends Exclude<BuilderUserFormFields, 'fla
             }
           })
         }}
+        onResubscribe={async formData => {
+          const selectedMemberplan = filteredMemberPlans.data?.memberPlans.nodes.find(
+            mb => mb.id === formData.memberPlanId
+          )
+          await resubscribe({
+            variables: formData,
+            onCompleted() {
+              window.location.href = selectedMemberplan?.confirmationPage?.url ?? ''
```

Comment:
I think the URL should be returned by the resubscribe action, and the onCompleted action should probably in the declaration above.